02-set-up.Rmd

```diff
@@ -46,9 +46,9 @@ This is partly facilitated by CRAN tests which ensure that
 R packages work on all OSs mentioned above.
 There are some operating system-specific quirks that may influence the choice
 of OS and how it is set-up for R programming in the long-term.
-Basic system information can be queried from within R using `Sys.info()`, as illustrated below for a selection of the functions output elements:
+Basic system information can be queried from within R using `Sys.info()`, as illustrated below for a selection its output:
 
-```{r}
+```{r, eval=FALSE}
```

Comment:
As an aside, I tend to use `eval=TRUE, results="hide"` to avoid typos within R code.